src/client/src/App/Credit/SellSide/SellSideTradeTicket/SellSideTradeTickerFooter.tsx

```diff
@@ -92,6 +92,7 @@ const [quoteRequest$, sendQuote] =
   createSignal<{ rfqId: number; dealerId: number }>()
 quoteRequest$
   .pipe(
+    tap(console.log),
```

Comment:
Was this left by mistake?

Reply:
Yes it was, I found a but with the SellSide ticket and was trying to debug it but I had forgotten it Thank you